src/components/TradeWidget/PriceEstimations.tsx

```diff
@@ -152,19 +233,12 @@ const OnchainOrderbookPriceEstimation: React.FC<OnchainOrderbookPriceEstimationP
   }
   const displayPrice = price === 'Infinity' || invertedPrice === 'Infinity' ? 'N/A' : price
 
-  const displayBaseToken = isPriceInverted ? quoteToken : baseToken
-  const displayQuoteToken = !isPriceInverted ? quoteToken : baseToken
-
-  const displayBtName = displayTokenSymbolOrLink(displayBaseToken)
-  const displayQtName = displayTokenSymbolOrLink(displayQuoteToken)
-
   return (
     <>
       <span>
-        <HighlightedText>Onchain orderbook price</HighlightedText> <HelpTooltip tooltip={OnchainOrderbookTooltip} /> for
-        selling{' '}
+        <HighlightedText>{!isPriceInverted ? 'Selling' : 'Buying'}</HighlightedText>{' '}
         <strong>
-          {+amount || '1'} {displayTokenSymbolOrLink(quoteToken)}
+          {+amount || '1'} {displayTokenSymbolOrLink(!isPriceInverted ? quoteToken : baseToken)}
```

Comment:
When price is inverted, the amount should be inverted as well, otherwise it won't make sense.

Example:
![screenshot_2020-08-14_08-22-00](https://user-images.githubusercontent.com/43217/90265307-401cfb80-de07-11ea-97ad-3faca8c3c731.png)
![screenshot_2020-08-14_08-22-07](https://user-images.githubusercontent.com/43217/90265306-3f846500-de07-11ea-858c-0ca4d1920610.png)
*Buying* amount should be `1/selling amount`

Reply:
Also, we should now remove the pair and leave only the quote token, such as for selling in the first example, we'll have:

`Selling 1 DAI  1.02... USDC`